corehq/apps/locations/tasks.py

```diff
@@ -170,6 +186,19 @@ def import_locations_async(domain, file_ref_id):
         for datasource in datasources:
             rebuild_indicators_in_place.delay(datasource.get_id)
 
+    if should_log_progress:
+        log.info("location import total time: %s", datetime.now() - start)
+        log.info(
+            "import_locations_async %s results: %s -> success=%s",
+            file_ref_id,
+            " ".join(
+                "%s=%r" % (name, getattr(results, name))
+                for name in ["messages", "warnings", "errors"]
+                if getattr(results, name)
+            ),
+            results.success,
+        )
```

Comment:
Is there another option here?  This will be nice for working on imports locally, but I'm not crazy about all the cruft this adds to this function.

Reply:
Reduced the cruft here by moving the progress logging part into the importer, which is nice because it will now apply anywhere an importer is used. 365889070a6f309d7b703049f6a8e29161ac2422

I didn't see a better place for logging task results, so I left that part here. I've implemented something similar to this several times in the past when doing local testing, and it always takes me a while to figure out how to get the results. The only thing that shows up in the browser, regardless of task success or failure, is

### Upload failed! Details:

with no details.

Reply:
Looks good, thanks!